webtau-core/src/test/groovy/com/twosigma/webtau/data/table/RecordTest.groovy

```diff
@@ -39,4 +39,24 @@ class RecordTest {
         def record = new Record(new Header(["c1", "c2", "c3"].stream()), ["v1", "v2", "v3"].stream())
         assert record.key == null
     }
+
+    @Test
+    void "should return defaultValue if a columnName is not present"() {
+        def record = new Record(new Header(["c1", "c2", "c3"].stream()), ["v1", "v2", "v3"].stream())
+        def valueC1 = record.get("c1", 42)
+        def valueC4 = record.get("c4", 42)
```

Comment:
or `get("noSuchColumn", 42)`?

Reply:
Intention here is to show more like columns that can be there but not in this case. I.e optional table columns. But I see what you mean. Sometimes I do that.